### PR TITLE
4.x Doc links update

### LINKED ISSUE #5405: 4.x Doc links update
Update doc links in README for all 4.x packages: 
as current link used is invalid  http://web3js.readthedocs.io/en/4.0/

### PR DESCRIPTION
**Pull request by avkos**
## Description

Fixes #5405
Please include a summary of the changes and be sure to follow our [Contribution Guidelines](../blob/1.x/CONTRIBUTIONS.md).

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist for 1.x:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run dtslint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:cov` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` with success.
- [ ] I have tested the built `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.

## Checklist for 4.x:

- [ ] I have selected the correct 4.x base branch.
- [ ] Within the description, the feature or issue is discussed in detail or an issue is linked.   
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added any required tests for the changes I made 
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `yarn` successfully
- [ ] I ran `yarn lint` successfully
- [ ] I ran `yarn build:web` successfully
- [ ] I ran `yarn test:unit` successfully
- [ ] I ran `yarn test:integration` successfully
- [ ] I ran `compile:contracts` successfully
- [ ] I have tested my code.
- [ ] I have updated the corresponding `CHANGELOG.md` file in the packages I have edited.

### CODE REVIEW

**Review by Spacesai1or — APPROVED** [review]

File: packages/web3-core/README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ yarn add web3-core
 | test:integration | Uses `jest` to run tests under `/test/integration` |
 | test:unit        | Uses `jest` to run tests under `/test/unit`        |
 
-[docs]: http://web3js.readthedocs.io/en/4.0/
+[docs]: https://docs.web3js.org/
 [repo]: https://github.com/ChainSafe/web3.js/tree/4.x/packages/web3-core
```
Comment (Muhammad-Altabba) [inline packages/web3-core/README.md]:
```suggestion
[repo]: https://github.com/web3/web3.js/tree/4.x/packages/web3-core
```

File: packages/web3-errors/README.md [hunk 2]
```diff
@@ -53,7 +53,7 @@ yarn add web3-errors
 | test:integration | Uses `jest` to run tests under `/test/integration` |
 | test:unit        | Uses `jest` to run tests under `/test/unit`        |
 
-[docs]: http://web3js.readthedocs.io/en/4.0/
+[docs]: https://docs.web3js.org/
 [repo]: https://github.com/ethereum/web3.js
```
Comment (Muhammad-Altabba) [inline packages/web3-errors/README.md]:
```suggestion
[repo]: https://github.com/web3/web3.js/tree/4.x/packages/web3-errors
```

File: packages/web3-core/README.md [hunk 3]
```diff
@@ -53,7 +53,7 @@ yarn add web3-core
...
 [npm-image]: https://img.shields.io/npm/v/web3-core-method.svg
```
Comment (Muhammad-Altabba) [inline packages/web3-core/README.md]:
```suggestion
[npm-image]: https://img.shields.io/npm/v/web3-core.svg
```

File: packages/web3-errors/README.md [hunk 4]
```diff
@@ -53,7 +53,7 @@ yarn add web3-errors
...
 [npm-image]: https://img.shields.io/npm/v/web3-core-method.svg
```
Comment (Muhammad-Altabba) [inline packages/web3-errors/README.md]:
```suggestion
[npm-image]: https://img.shields.io/npm/v/web3-errors.svg
```

**Review by Muhammad-Altabba — APPROVED** [review]
It would be great to update the `repo` and `npm-image` at all README.md files (I added the suggestion for just 2. But this applies to all packages).
And it would be greater if the scripts were also updated:smile:.
Thanks,

**Review by jdevcs — COMMENTED** [review]
lgtm, except required changes proposed by Muhammad

File: packages/web3-core/README.md (hunk 1 above)
Comment (avkos) [reply packages/web3-core/README.md]: done

File: packages/web3-errors/README.md (hunk 2 above)
Comment (avkos) [reply packages/web3-errors/README.md]: done

**Review by jdevcs — APPROVED** [review]

**Pull request merged**